PrReviewCode/Program.cs

```diff
@@ -4,6 +4,8 @@ public class Program
 {
     public static void Main(string[] args)
     {
+        throw new Exception("Test Exception");
```

Comment:
Test exception with message 'Test Exception' should not be committed to production code.

 **SUGGESTION:**

```suggestion
 // Remove this test exception or add conditional check to prevent accidental deployment
```